facebook-instant-articles.php

```diff
@@ -342,20 +334,47 @@ function inject_url_claiming_meta_tag() {
 		$publishing_settings = Instant_Articles_Option_Publishing::get_option_decoded();
 		$fb_page_settings = Instant_Articles_Option_FB_Page::get_option_decoded();
 
-		if ( isset( $fb_page_settings['page_id'] ) ) {
+		if ( isset( $fb_page_settings[ 'page_id' ] ) ) {
 			?>
-			<meta property="fb:pages" content="<?php echo esc_attr( $fb_page_settings['page_id'] ); ?>" />
+			<meta property="fb:pages" content="<?php echo esc_attr( $fb_page_settings[ 'page_id' ] ); ?>" />
 			<?php
 		}
 	}
 	add_action( 'wp_head', 'inject_url_claiming_meta_tag' );
 
+	/**
+	 * Automatically add meta tag for Instant Articles Open Publish scraper
+	 *
+	 * @since 4.0
+	 */
+	function inject_op_markup_meta_tag() {
+		$post = get_post();
+		// Transform the post to an Instant Article.
+		$adapter = new Instant_Articles_Post( $post );
+		$url = $adapter->get_canonical_url();
+		$url = add_query_arg( 'op', '1', $url );
+		?>
+		<meta property="ia:markup_url" content="<?php echo esc_attr( $url ); ?>" />
+		<?php
+	}
+	add_action( 'wp_head', 'inject_op_markup_meta_tag' );
+
 	// Initialize the Instant Articles meta box.
 	Instant_Articles_Meta_Box::init();
 
-	// Initialize the Instant Articles publisher.
-	Instant_Articles_Publisher::init();
+	function op_markup_version( ) {
+		$post = get_post();
+
+		if (isset($_GET['op']) && $_GET['op']) {
```

Comment:
2 things:
- Formating nit, should be `if ( isset( $_GET[ 'op' ] ) && $_GET[ 'op' ] ) {`
- Same thing about "op" usage. Maybe we should drop this OP from everywhere.